processor/agentmetricsprocessor/config.go

```diff
@@ -19,4 +19,8 @@ import "go.opentelemetry.io/collector/config"
 // Config defines configuration for Resource processor.
 type Config struct {
 	config.ProcessorSettings `mapstructure:",squash"`
+
+	// BlankMetrics is a list of metrics that need a label called
+	// "blank" with an empty value.
```

Comment:
This seems like a hack. Would this even work if you ever wanted to add multiple labels with blank values?
Is this done solely to avoid a fork? Are we expecting to undo this as soon as upstream allows blank values in the `add_label` transform?

Reply:
Of course this is a hack. It's a much smaller hack than the previous hack. This will go away as soon as `add_label` is fixed upstream, just like most of the other things in agentmetricsprocessor.